shard.yml

```diff
@@ -4,20 +4,17 @@ version: 0.6.0
 authors:
   - robacarp
 
-crystal: 0.32.1
+crystal: 0.33.0
 
 license: MIT
 
 dependencies:
   redis:
     github: stefanwille/crystal-redis
-    version: ~> 2.5.0
```

Comment:
These shouldn't be removed.  They indicate the minimum requires versions mosquito requires to work.  Removing them would allow someone to install mosquito with an unsupported version of redis.